### DISCUSSION TITLE
Help deleting pull requests

### DISCUSSION THREAD

**Discussion by felipe-pace-evaristo** [community/community#28914] [Pull Requests] [answered]
Hi everyone! I need help getting 3 pull requests deleted. I've tried to reach out to GitHub Support many times, to no avail. Does anyone one if there's any way I can do it myself or any communication channel with GitHub that is actually checked?

**Comment by ghost** [comment]
No, GitHub policy is not to remove pull requests. You can write to them but you must have a very good reason to have you PR deleted by them.

**Reply by welas234** [reply]
all pull remove

**Reply by Tinabates** [reply]
Scammer trying to steal just making sure he is blocked and wasn’t able to change administrative rights or administrator

**Reply by Tinabates** [reply]
It’s fine long as he hasn’t taken my rights and my tokens

**Comment by ihnatenkoo** [comment, marked as answer]
You can't delete a delete request manually - you and the owner of the repo (and all users with forced access to it) can close it, but it will remain in the log. It's part of the philosophy of not denying/hiding what happened during development.

However, if there are good reasons for removing it (it's basically a violation of the Github Terms of Service), Github Support will remove it for you.

**Comment by felipe-pace-evaristo** [comment]
Will it go away if I delete my account?

**Reply by airtower-luna** [reply]
No. In that case it'll be reassigned to the "ghost" user. :ghost:

**Comment by felipe-pace-evaristo** [comment]
I opened a ticket with Support 2+ days ago. I wonder how long it’ll take them to get back to me…

**Comment by Remilekun-Odegbami** [comment]
I have this exact same problem. I don't even know what to do

**Reply by felipe-pace-evaristo** [reply]
@Remilekun-Odegbami, Support eventually got back to me and helped me delete them. I suggest opening a ticket with them and being patient...

**Reply by Remilekun-Odegbami** [reply]
How do I open a ticket?

**Reply by airtower-luna** [reply]
The contact form is here: https://support.github.com/request

**Reply by Remilekun-Odegbami** [reply]
Thank you. The Virtual assistant said I cannot request for the pull request to be deleted since it is not my repo (it is a forked repo). At this juncture, it seems there is nothing I can do about the pull request, I will let it be. Thanks once again for your assistance @felipepacci and @airtower-luna.

**Comment by felipe-pace-evaristo** [comment]
Who is the scammer? What user?



*Felipe Fabricio Pacci Evaristo*

*Mobile:* +1 (773) 569-2978

*Email:* ***@***.***

*Skype:* felipepacci

*Pronouns:* he/him/his
